rmw_connextddsmicro/package.xml

```diff
@@ -19,5 +19,10 @@
 
   <export>
     <build_type>ament_cmake</build_type>
+    
+    <deprecated>
+        The Connext Micro RMW, rmw_connextddsmicro, will be removed in a future ROS 2 distro. Use
+        the Connext Pro RMW, rmw_connextdds, instead.
```

Comment:
according to https://github.com/ros/rosdistro/blob/master/rolling/distribution.yaml, this package has bever released, and user can only use this package with source build?

```suggestion
        This package will be removed in a future ROS 2 distro. Use rmw_connextdds, instead.
```

Reply:
Yeah, it seems like it's never been released, but I prefer to just deprecate it for now and target a removal of it for the L release cycle. There seems to be people out here using it.